### ISSUE TITLE
Overused expression should not count primitives

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#1152]
# Bug report

## What's wrong

Currently, this is an error in our linter:

```
  359:8    WPS204 Found overused expression: []; used 8 times
  print(*[], **{'@': 1})  # noqa: WPS517, WPS445
```

But, how can one remove this violation? Impossible.
We should ignore single node collections and named variables.

**Comment by sobolevn** [comment]
The same is applied to `set()`, `list()`, `factory()` an other simple callables.

**Comment by memery-imb** [comment]
Thanks for doing this. I was about to post the exact same issue!

**Comment by memery-imb** [comment]
Just sent over $10 as a token of my appreciation. Keep up the great work!

**Comment by sobolevn** [comment]
@memery-imb thank you! That's awesome!

I feel so good from knowing that people actually use and benefit from this project! 👍

**Comment by pawelarybak** [comment]
Should it count minus unary operator (`-`)? In my code i often use negative value of a named variable and such usage breaks this rule.

Example:
```python
my_value = 5
position = (0, -my_value)
```

**Comment by sobolevn** [comment]
@pawelarybak yes, it should not. Please, create a new issue.
